input/templates/partials/myaccount/my-account-sidebar.hbs

```diff
@@ -1,29 +1,29 @@
 <!-- start myaccount/my-account-sidebar.hbs -->
 <div class="my-account-sidebar">
-  <div {{#if positionOne}}id="my-account-mobile-content"{{/if}} class="my-account-sidebar-items {{#if positionOne}}active{{/if}}">
+  <div {{#if myPersonalDetailsTab}}id="my-account-mobile-content"{{/if}} class="my-account-sidebar-items {{#if myPersonalDetailsTab}}active{{/if}}">
```

Comment:
I have renamed all these flags to know which tab is active. The numeric naming is very limited because when you want to add a new tab in the middle you need to rename all the following tabs. Really bad choice of name. So with this new name, it is completely independent on how many other tabs there are and you can organize the tabs easily.